src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -377,5 +395,15 @@ public bool OnDeepLink(string deepLink)
             handler(deepLink);
             return true;
         }
+
+        public void OnInboxUpdated()
+        {
+            //Adding Inbox updated listener
+            EventHandler handler = OnMessageCenterUpdated;
```

Comment:
An easier way to write this in one line:

`OnMessageCenterUpdated?.Invoke(this, EventArgs.Empty)`